bitwarden_license/src/Portal/Views/Sso/Index.cshtml

```diff
@@ -134,6 +134,30 @@
                     </div>
                 </div>
             </div>
+            <div class="row">
+                <div class="col-7 form-group">
+                    <label asp-for="Data.AdditionalScopes">@i18nService.T("AdditionalScopes")</label>
```

Comment:
I don't believe its necessary to use the `i18nService` to translate the label names here. It should automatically use the translated `Display(Name="...")` assigned on the ViewModel.

Reply:
Hey that worked! Nice!